js/util/counter_clockwise.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+// http://bryceboe.com/2006/10/23/line-segment-intersection-algorithm/
+module.exports = function (a, b, c) {
+    return (c.y - a.y) * (b.x - a.x) > (b.y - a.y) * (c.x - a.x);
+};
```

Comment:
I don't think we need to cut out small functions like this into a separate file. Maybe move to `util.js`? Or maybe we should have `geom_util.js` for all functions like this.